MVP_Clicker/Assets/Project/Scripts/Game/Areas/Monster/View/IMonsterView.cs

```diff
@@ -1,14 +1,19 @@
 using System;
+using System.Collections.Generic;
 using UnityEngine;
 
 namespace Project.Scripts.Game.Areas.Monster.View
 {
     public interface IMonsterView
     {
         public event Action Damaged;
-        
+
         void SetCurrentHp(int currentHp);
 
         public void SetMonsterImage(Sprite monsterImage);
+
+        public void SetRandomMonsterImage();
```

Comment:
You named this method 'set'. In such methods I expect to set some value but in your case there is not parameter. 
So best is to rename to another name. May be `UpdateImageRandomlyFromPull`